src/main/scala/beam/agentsim/infrastructure/ZonalParkingManager.scala

```diff
@@ -138,7 +138,7 @@ object ZonalParkingManager extends LazyLogging {
     )
     ZonalParkingManager[GEO](
       vehicleManagerId,
-      parking.zones.toMap,
+      parking.zones.filter(_._2.chargingPointType.isEmpty).toMap,
```

Comment:
To verify, you want only the ones with an empty charging point type? I ask because filter is a tricky keyword I always get mixed up for being filter out - versus its true nature of filter in.

Reply:
Yes, in the parking network classes we will track only stalls without charging point while in charging network only the ones with charging point